ObjCCBOR/CBORRepresentable.h

```diff
@@ -0,0 +1,19 @@
+//
+//  CBORRepresentable.h
+//  ObjCCBOR
+//
+//  Created by Hamilton Chapman on 30/04/2020.
+//  Copyright © 2020 Dash. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol CBORRepresentable
+
+- (nullable NSObject *)getCBORRepresentation;
```

Comment:
```suggestion
- (nullable NSObject *)CBORRepresentation;
```

This was bugging me for a while and (although I suggested it) I've finally worked out why. Working in so many languages at once can really scramble your brain 🥴

> If the method returns an attribute of the receiver, name the method after the attribute. The use of “get” is unnecessary, unless one or more values are returned indirectly.

[Coding Guidelines for Cocoa](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html)